src/view/com/composer/Composer.tsx

```diff
@@ -398,16 +403,48 @@ export const ComposePost = ({
           langs: toPostLanguages(langPrefs.postLanguage),
         })
       ).uris[0]
+
+      /*
+       * Wait for app view to have received the post(s). If this fails, it's
+       * ok, because the post _was_ actually published above.
+       */
```

Comment:
This will get called by prod code too, not just v2 code, so should think hard on if that's OK or now